packages/@webex/plugin-cc/test/unit/spec/services/core/aqm-reqs.ts

```diff
@@ -1,7 +1,9 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import AqmReqs from '../../../../../src/services/core/aqm-reqs';
 import HttpRequest from '../../../../../src/services/core/HttpRequest';
+import {WebSocketManager} from '../../../../../src/services/core/WebSocket/WebSocketManager';
```

Comment:
I had to add couple of more tests so coverage went above 85%. Discussed this with Ravi and I got it working... it has made the PR a little huge, however.